packages/backend/src/clients/github/Github.ts

```diff
@@ -257,16 +281,18 @@ export const createPullRequest = async ({
     head,
     base,
     installationId,
+    token,
 }: {
     owner: string;
     repo: string;
     title: string;
     body: string;
     head: string;
     base: string;
-    installationId: string;
+    installationId?: string;
+    token?: string;
 }) => {
-    const octokit = getOctokitRestForApp(installationId);
+    const { octokit, headers } = getOctokit(installationId, token);
```

Comment:
on createPullRequest we want to create it as a bot if installationId is avialable.